api/services/Slack.js

```diff
@@ -0,0 +1,32 @@
+var format = require('util').format
+var Promise = require('bluebird')
+var request = require('request')
+var post = Promise.promisify(request.post)
+
+module.exports = {
+  textForNewPost: function (post, community) {
+    var relatedUser, creator = post.relations.creator
+
+    if (post.isWelcome()) {
+      relatedUser = post.relations.relatedUsers.first()
+      return format('<%s|%s> joined <%s|%s>',
```

Comment:
Actually, in this case, I like it with format better because it makes the full message clearer right away when you read it, versus having to parse out all the higher complexity variable names inline